curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -62,12 +62,9 @@ int DiskCacheManager::Init(S3Client *client,
     }
     // start aync upload thread
     cacheWrite_->AsyncUploadRun();
-    // upload all file in write cache to S3
-    ret = cacheWrite_->UploadAllCacheWriteFile();
-    if (ret < 0) {
-        LOG(ERROR) << "upload cache write file error. ret = " << ret;
-        return ret;
-    }
+    std::thread th =
```

Comment:
kill curve-fuse need wait this thread exit. but th is local variable.

Reply:
no matter if curve-fuse be killed